### DISCUSSION TITLE
How do I recommend improvements in spacy models NER extraction?

### DISCUSSION THREAD

**Discussion by nitinthewiz** [explosion/spaCy#11019] [Help: Model Advice] [answered]
Hi,

I'm using en_core_web_md and I see a few discrepancies in the NERs it extracts from a text.
For example - 
"Binance" is marked as a GPE while it should be ORG
"Coinbase" is marked a PERSON when it should be ORG

How can I suggest these improvements to the models? Is opening a discussion the best way?

Or should I collect these locally and train the model further using something like augmenty to get the correct labels?

I understand that entity recognition is only about 85% accurate in these models and 90% in the transformer model.

**Comment by xxyzz** [comment, marked as answer]
You could use the [Entity Ruler](https://spacy.io/usage/rule-based-matching#entityruler) and add it before the `ner` component:

```python
ruler = nlp.add_pipe("entity_ruler", before="ner")
```

**Reply by nitinthewiz** [reply]
Thank you for pointing me in this direction, which seems to be more native than using a third party tool. I'd still like to know if we can contribute to improving the models, though.

**Comment by polm** [comment]
We don't collect training data from users to directly improve the models, or to address specific issues. If the models are performing poorly for your use case or entities you care about you should train your own model. It's also our expectation that while the pretrained models are broadly useful and great for getting started, for serious applications you should usually be training your own model for your own data. 

Also see #3052 about inaccurate predictions in general.
